script.module.tmdbsimple/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.tmdbsimple"
+       name="tmdbsimple"
+       version="2.2.0+matrix.1"
+       provider-name="Celia Oakley, rmrector">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.requests" version="2.15.1"/>
+  </requires>
+  <extension point="xbmc.python.module"
+             library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">A wrapper for The Movie Database API v3</summary>
+    <description lang="en_GB">Packed for Kodi from https://github.com/celiao/tmdbsimple</description>
+    <platform>all</platform>
+    <license>GPL-3.0</license>
+    <website>https://github.com/rmrector/script.module.tmdbsimple</website>
```

Comment:
website here looks like tmdb and not exactly your repo

Reply:
https://github.com/xbmc/repo-scripts/pull/1251#discussion_r360692039

What exactly do you want changed, and to what exactly?

Reply:
Disregard, it's right